config/global.ini.php

```diff
@@ -155,6 +159,7 @@
 ; NOTE: you must also set [log] log_writers[] = "screen" to enable the profiler to print on screen
 enable_sql_profiler = 0
 
+[Debug]
```

Comment:
Guess this one was added by accident
```suggestion
```

Reply:
Yes, not sure where that came from! Thanks for spotting it :slightly_smiling_face: